FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/ALLEGRO_o1_v03.xml

```diff
@@ -25,21 +25,23 @@
     <constant name="world_size" value="25*m"/>
     <constant name="world_x" value="world_size"/>
     <constant name="world_y" value="world_size"/>
-    <constant name="world_z" value="world_size"/>
+    <constant name="world_z" value="world_size"/>    
   </define>
 
   <include ref="./DectDimensions.xml" />
 
-  <include ref="Beampipe.xml"/>
+  <!--<include ref="Beampipe.xml"/>-->
```

Comment:
```suggestion
```

Reply:
should we remove also the Beampipe.xml file from the folder? Same for IDEA

Reply:
should I remove also HOMAbsorber.xml then?

Reply:
yes :+1:

Reply:
fixed